distributed/dashboard/scheduler.py

```diff
@@ -319,6 +322,34 @@ def get(self):
         self.set_header("Content-Type", "text/plain")
 
 
+class EventsHandler(WebSocketHandler):
+    def initialize(self, server=None, extra=None):
+        self.server = server
+        self.extra = extra or {}
```

Comment:
This appears to be unused.

Reply:
I assumed this was invoked somewhere. If I don't include this the `self.server` attribute doesn't exist.

Reply:
On a second read of this comment I'm not 100% sure what you are referring to.

Reply:
I meant `self.extra`.  I assumed by your previous comment that it actually was necessary though by some upstream class.

Reply:
Ah thanks! After further inspection, I'm not using this. It may be required by something else. I just copy/pasted it from somewhere else (although I can't seem to find where now).

The `extra` kwarg seems to be necessary, so I could alternatively not store it as an attribute, or wrap up the kwargs and optionally store those.

My inclination is to leave things as they are. Are you happy with that?